File: src/iniciante/Bee1020.java
package iniciante;

import java.util.Scanner;

public class Bee1020 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int idadeEmDias = input.nextInt();
        int resultado;
        int vetorQuantEmDias [] = {365, 30};

        for (int i = 0; i <= 1; i++) {
            resultado = idadeEmDias/vetorQuantEmDias[i];
            idadeEmDias %= vetorQuantEmDias[i];
            if (i == 0) {
                System.out.println(resultado + " ano(s)");
            } else {
                System.out.println(resultado + " mes(es)");
            }
        }
        System.out.println(idadeEmDias + " dia(s)");
    }
}

File: beginner/bee1020.py
def age_in_days(total_days):
    total_years = total_days // 365
    total_days = total_days % 365
    total_month = total_days // 30
    total_days = total_days % 30

    print(f'{total_years} ano(s)\n{total_month} mes(es)\n{total_days} dia(s)')

n = int(input())
age_in_days(n)
